tests/introspection/iOS/iOSApiSelectorTest.cs

```diff
@@ -801,7 +801,8 @@ protected override bool CheckStaticResponse (bool value, Type actualType, Type d
 				switch (declaredType.Name) {
 				case "CIImage":
 					if (Runtime.Arch == Arch.SIMULATOR)
-						return true;
+						// does not work (like other IOSurface API) in beta 1
+						return true; // return !TestRuntime.CheckXcodeVersion (11, 0);
```

Comment:
Should we create an issue for this before we forget?

Reply:
I'll file a rdar/feedback and add it to our list

Reply:
actually logic is wrong (it's not a `Skip` API) and it works. Updating PR